From 55b9602e9728e5c0cdfa7e907b5a24a75fe23283 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Fri, 13 Dec 2024 15:22:07 +0100
Subject: [PATCH 01/10] Subject: [PATCH 1/2] missing-plugins: Remove the
 mpegaudioversion field

From missing plugins requests as it's a duplicate of mpegversion
and its presence would break codec discovery when using RPM.

https://bugzilla.redhat.com/show_bug.cgi?id=680809
---
 .../gst-plugins-base/gst-libs/gst/pbutils/missing-plugins.c    | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/subprojects/gst-plugins-base/gst-libs/gst/pbutils/missing-plugins.c b/subprojects/gst-plugins-base/gst-libs/gst/pbutils/missing-plugins.c
index 69330b933b..c4fd3e6737 100644
--- a/subprojects/gst-plugins-base/gst-libs/gst/pbutils/missing-plugins.c
+++ b/subprojects/gst-plugins-base/gst-libs/gst/pbutils/missing-plugins.c
@@ -191,6 +191,9 @@ copy_and_clean_caps (const GstCaps * caps)
   gst_structure_remove_field (s, "play-speed");
   gst_structure_remove_field (s, "play-scale");
   gst_structure_remove_field (s, "dynamic_range");
+  /* mpegaudioversion is a duplicate for mpegversion
+   * https://bugzilla.redhat.com/show_bug.cgi?id=680809 */
+  gst_structure_remove_field (s, "mpegaudioversion");
 
   return ret;
 }
-- 
2.47.0

